Explorer/Assets/DCL/InWorldCamera/PhotoDetail/PhotoDetailInfoController.cs

```diff
@@ -0,0 +1,158 @@
+using Cysharp.Threading.Tasks;
+using DCL.AvatarRendering.Wearables;
+using DCL.AvatarRendering.Wearables.Helpers;
+using DCL.Backpack;
+using DCL.Browser;
+using DCL.Chat;
+using DCL.InWorldCamera.CameraReelStorageService;
+using DCL.InWorldCamera.CameraReelStorageService.Schemas;
+using DCL.InWorldCamera.ReelActions;
+using DCL.Multiplayer.Connections.DecentralandUrls;
+using DCL.Passport;
+using DCL.Profiles;
+using DCL.WebRequests;
+using ECS.SceneLifeCycle.Realm;
+using MVC;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading;
+using UnityEngine;
+using Utility;
+
+namespace DCL.InWorldCamera.PhotoDetail
+{
+    public class PhotoDetailInfoController : IDisposable
```

Comment:
Please provide a short description about the purpose of the controller and any detail you consider interesting for somebody that finds it for the first time.